pdm/pep517/base.py

```diff
@@ -145,6 +145,13 @@ def meta(self) -> Metadata:
             self._meta.validate(False)
         return self._meta
 
+    @property
+    def meta_version(self) -> str:
+        meta_version = self.meta.version
+        if meta_version is None:
+            return "0.0.0"
+        return to_filename(safe_version(meta_version))
```

Comment:
I first considered moving this to `safe_version`, but that is the only use of it now anyway.